src/EnergyPlus/FluidProperties.cc

```diff
@@ -2874,7 +2862,7 @@ namespace FluidProperties {
         if (!state.dataGlobal->WarmupFlag && ErrorFlag) {
             ++this->errors[(int)RefrigError::SatPress].count;
             // send warning
-            if (this->errors[(int)RefrigError::SatPress].count <= df->RefrigErrorLimitTest) {
+            if (this->errors[(int)RefrigError::SatPress].count <= state.dataFluidProps->RefrigErrorLimitTest) {
```

Comment:
[src/EnergyPlus/FluidProperties.cc:2854]:(style),[variableScope],The scope of the variable 'df' can be reduced.